automobile/scenario-creation-tutorial.md

```diff
@@ -1,65 +1,59 @@
 ## Scenario creation tutorial
 
 This small tutorial explains step-by-step how to create a scenario inspired from a real world map and then add traffic using SUMO.
+This tutorial has been written for `linux` and `macOS`, but the same can be achieved on `Windows` converting the following commands to `DOS`.
 
-### Download the map
+
+### Creation of the Webots project directory
+
+```sh
+export WBT_PROJECT_PATH=/your/webots/project/path  # define here your project path
+mkdir -p $WBT_PROJECT_PATH/worlds/map_net
+```
+
+
+### Download the OpenStreetMap map
 
 We will use a part of the OpenStreetMap map to generate the Webots world file. To download the map go to the [OpenStreetMap website](https://www.openstreetmap.org/export). From there you can select the part of the map you want and download it:
 
 %figure "Export a part of map from OpenStreetMap"
 ![osm_export.png](images/osm_export.png)
 %end
 
-### Generate the Webots world
-
-As explained in the [OpenStreetMap importer](openstreetmap-importer.md) section, you should use the previously downloaded map to generate the Webots world.
-
-> **Note**:
-This importer uses splines to improve and smooth the path of the roads, unfortunately the OpenStreetMap to SUMO importer does not support this, it is therefore recommended to disable it (setting the spline subdivision to 0) if you want then to add traffic using SUMO.
 
-> **Note**:
-It is strongly recommended to not use the 3D feature of the [OpenStreetMap importer](openstreetmap-importer.md) otherwise it will not be possible to add traffic using SUMO.
+### Get the map projection
 
-Once the conversion is complete, the importer will display the number of objects generated, the map offset, the reference coordinates and the projection used:
+Get the map projection parameters which will be reuse them to generate the SUMO network file.
 
+```sh
+cd $WEBOTS_HOME/projects/automobile/resources/OSM_importer
+python importer.py --input=/path/to/map.osm --extract-projection
```

Comment:
For consistency, we should probably keep using `myMap.osm` here instead of `map.osm`.